### PR TITLE
Added reference to Secrets Manager section in the lambda docs

### PR DESCRIPTION
**Pull request by AlexanderWert**

### CODE REVIEW

File: docs/serverless.asciidoc [hunk 1]
```diff
@@ -54,6 +54,8 @@ The APM Lambda Extension and the APM Python agent are configured through environ
 For the minimal configuration, you will need the _APM Server URL_ to set the destination for APM data and an _{apm-guide-ref}/secret-token.html[APM Secret Token]_.
 If you prefer to use an {apm-guide-ref}/api-key.html[APM API key] instead of the APM secret token, use the `ELASTIC_APM_API_KEY` environment variable instead of `ELASTIC_APM_SECRET_TOKEN` in the following configuration.
 
+Alternatively, you can {apm-guide-ref}/aws-lambda-secrets-manager.html[use the AWS Secrets Manager to store your APM authentication key] (i.e. the _APM Secret Token_ or the _APM API key_) instead of providing the secret value as plaintext in the environment variables.
```
Comment (beniwohli) [inline docs/serverless.asciidoc:57]: Would this be the recommended method to store the secret key in Lambda deployments? In that case, I'd reword the whole section to steer the user towards this instead of just suggesting it as an alternative.

If not, LGTM :)

**Review by beniwohli — APPROVED** [review]

File: docs/serverless.asciidoc (hunk 1 above)
Comment (AlexanderWert) [inline docs/serverless.asciidoc:57]:
```suggestion
For production environments, we recommend {apm-guide-ref}/aws-lambda-secrets-manager.html[using the AWS Secrets Manager to store your APM authentication key] instead of providing the secret value as plaintext in the environment variables.
```
